src/collection/collectionmodel.cpp

```diff
@@ -94,6 +96,7 @@ CollectionModel::CollectionModel(CollectionBackend *backend, Application *app, Q
       album_icon_(IconLoader::Load("cdcase")),
       playlists_dir_icon_(IconLoader::Load("folder-sound")),
       playlist_icon_(IconLoader::Load("albums")),
+      icon_cache_(new QNetworkDiskCache(this)),
```

Comment:
This needs to be static since we use 7 instances of the collection model, look at core/network.cpp